x-pack/plugins/security_solution/public/detections/components/callouts/need_admin_for_update_callout/index.tsx

```diff
@@ -33,20 +34,24 @@ const needAdminForUpdateRulesMessage: CallOutMessage = {
  * hasIndexManage is also true, then the user should be performing the update on the page which is
  * why we do not show it for that condition.
  */
-const NeedAdminForUpdateCallOutComponent = (): JSX.Element => {
+const NeedAdminForUpdateCallOutComponent = (): JSX.Element | null => {
   const [{ signalIndexMappingOutdated, hasIndexManage }] = useUserData();
 
   const signalIndexMappingIsOutdated =
     signalIndexMappingOutdated != null && signalIndexMappingOutdated;
 
   const userDoesntHaveIndexManage = hasIndexManage != null && !hasIndexManage;
-
-  return (
-    <CallOutPersistentSwitcher
-      condition={signalIndexMappingIsOutdated && userDoesntHaveIndexManage}
-      message={needAdminForUpdateRulesMessage}
-    />
-  );
+  const shouldShowCallout = signalIndexMappingIsOutdated && userDoesntHaveIndexManage;
+
+  return shouldShowCallout ? (
+    <>
+      <CallOutPersistentSwitcher
+        condition={shouldShowCallout}
```

Comment:
Would this ever receive `false`? Also I think the only place `CallOutPersistentSwitcher` is used is here. I think we could remove the `condition` prop right?

Reply:
Nope, fixed it. Thanks!